### DISCUSSION TITLE
Projects not supported in mobile app when connected to enterprise?

### DISCUSSION THREAD

**Discussion by IdanAdar** [community/community#136104] [Mobile]
When using the Mobile app and connecting to an Enterprise server, the projects option is not available. Is this truly the case or just an admin option that is not enabled?

**Comment by vimukthijayasanka** [comment]
To clarify this, you can:

Check with your system administrator to see if the "Projects" feature is enabled.
Ensure that you are using the latest version of the mobile app.
Verify that your user role has the necessary permissions to access "Projects."

**Reply by IdanAdar** [reply]
I am using the latest mobile app version. We do have Projects enabled in our GHE 3.9 installation and I can use it fully, if not using the mobile app.
